detrex/config/configs

```diff
@@ -0,0 +1 @@
+/mnt/lustre/zhulei1/GitRepo/detrex/configs
```

Comment:
Hi~ this seems like a specific submodule linked to your local projects, if this is needed or not~